WordPress/src/main/java/org/wordpress/android/designsystem/DesignSystemTheme.kt

```diff
@@ -0,0 +1,27 @@
+package org.wordpress.android.designsystem
+
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.CompositionLocalProvider
+import androidx.compose.runtime.ReadOnlyComposable
+import androidx.compose.runtime.remember
+
+@Composable
+fun DesignSystemTheme(
```

Comment:
Appreciate your efforts in creating an independent theme here.  We would rather still use MaterialTheme to take advantage of many of its features like here in [AppTheme.kt](https://github.com/wordpress-mobile/WordPress-Android/blob/aadadca0f1ea225179b0fe28eb6cb36f53e0662e/WordPress/src/main/java/org/wordpress/android/ui/compose/theme/AppTheme.kt#L38), and here in [Theme.kt](https://github.com/wordpress-mobile/WordPress-Android/blob/aadadca0f1ea225179b0fe28eb6cb36f53e0662e/WordPress/src/main/java/org/wordpress/android/ui/domains/management/Theme.kt#L40)

Reply:
Thanks for taking a look @ravishanker. I also had the same concern but thought an independent theme would make sense since we have custom definitions for all 3 systems within Material Theme (`colors`, `typography`, and `shapes`). 
 
Another reason I implemented a custom `colors` system was because the default attributes that come with Material Theme don't correlate with what we have defined [here](https://www.figma.com/file/LyzJJa6ANKToUswWmRrZee/%F0%9F%A7%B1-Jetpack-App-Design-System-iOS?type=design&node-id=102-186&mode=design&t=UNkEMaJRRKFoKbdC-0). From what I understand, if we extended Material Theme's colors, we would have to define the `surface` and `background` attributes which don't currently exist in the DS definition.

I can update the colors to [extend Material Theme](https://developer.android.com/jetpack/compose/designsystems/custom#extending-material) instead of replacing the `colors` system, but that would mean the attributes that come with Material Theme would need to be defined too. In this case what would we define `surface` and `background` colors to be, for example?

Reply:
Use whatever is the material default for surface, and background. `MaterialTheme.colors.background`

The design system is just a guide line.  It is meant to be adapted to each platform.